package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "community",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
```

Comment:
(note to myself)

opened this up in a codespace,
looks like we need to add `next export` to this line, though that led to this error:

```
    The "next export" command has been removed in favor of "output: export" in next.config.js. Learn more: https://nextjs.org/docs/advanced-features/static-html-export
```

will come back to this

https://nextjs.org/docs/pages/building-your-application/deploying/static-exports#configuration